common/ledger/blkstorage/blockindex.go

```diff
@@ -248,6 +252,44 @@ func (index *blockIndex) getTXLocByBlockNumTranNum(blockNum uint64, tranNum uint
 	return txFLP, nil
 }
 
+func (index *blockIndex) exportUniqueTxIDs(writer io.Writer) error {
+	if !index.isAttributeIndexed(IndexableAttrTxID) {
+		return ErrAttrNotIndexed
+	}
+	writer.Write([]byte{exportFileFormatVersion})
```

Comment:
The following is not that important for now -- while looking at the test, I realized that we don't prepend length with the version. We are good as long as we use single-digit format (8 more format changes to go -- kind of rare too).

Reply:
254, not 8, more to go :-)

Reply:
ah. got it. 8 bits. in the worst case, we still have 63 more years 😄 to go (assuming we change the format every minor release).